docs/admin-guide/configuration.md

```diff
@@ -30,7 +30,9 @@ You must create a webhook for each repository you want to receive notifications
 2. Select **Add Webhook**.
 3. Set the following values:
    * **Title:** `Mattermost Bitbucket Webhook - <repository_name>`, replacing `repository_name` with the name of your repository.
-   * **URL:** `https://your-mattermost-url.com/plugins/bitbucket/webhook`, replacing `https://your-mattermost-url.com` with your Mattermost deployment's Site URL.
+   * **URL:** `https://your-mattermost-url.com/plugins/bitbucket/webhook/your-webhook-secret`
+      * replace `https://your-mattermost-url.com` with your Mattermost deployment's Site URL.
+      * replace `your-webhook-secret` with the secret generated in System Console > Plugins > Bitbucket > Webhook Secret
```

Comment:
```suggestion
      * replace `your-webhook-secret` with the secret generated in System Console > Plugins > Bitbucket > Webhook Secret.
```

Reply:
Let's add the period to be consistent with the other lines. Thanks!

Reply:
Done